cfgov/jinja2/v1/_includes/molecules/notification.html

```diff
@@ -24,19 +24,21 @@
 {% macro render(type, is_visible, message, explanation=none) %}
 
   {% set type_lookup  = {
-        'success' : 'approved',
-        'warning' : 'error',
-        'error'   : 'delete',
+        'default': 'information-round',
+        'success': 'approved-round',
+        'warning': 'warning-round',
+        'error':   'error-round',
       }
   %}
   {% set icon = type_lookup[type]  %}
-  {% set type = ('m-notification__' + type | string) if type else '' %}
+  {% set type = ('m-notification__' + type | string)
+     if type and type != 'default'
+     else '' %}
```

Comment:
I suppose it's more descriptive to have all the states in the `type_lookup`, but looks like you could just place the default here and avoid an extra if check:
```
if type
else 'information-round' %}
```

Either way is fine. Trades off either way.

Reply:
@anselmbradford I'm not sure I'm following. There's a type class as well as icon changes. This part is creating the appropriate class name. Now that I review it though, I'm not sure we want to skip the class just because it's `default`. @Scotchester do you remember what we were thinking here?

Reply:
Oh wait, I was confused on what was being written into the type. The default will always be an empty class, doh!

Reply:
Yeah, which looking back seems incorrect. I think we just need to pull out the `!=`.